src/drivers/wayland/lv_wayland_smm.h

```diff
@@ -0,0 +1,67 @@
+/**
+ * @file lv_wayland_smm.h
+ *
+ */
+#ifndef LV_WAYLAND_SMM_H
+#define LV_WAYLAND_SMM_H
+#include <stddef.h>
+#include <stdbool.h>
```

Comment:
```suggestion
#include LV_STDDEF_INCLUDE
#include LV_STDBOOL_INCLUDE
```
to fix the uefi CI issue.

Reply:
Thanks for the help, this one wasn't so obvious.